rest/src/main/scala/io/udash/rest/openapi/OpenApiMetadata.scala

```diff
@@ -52,6 +52,16 @@ final case class OpenApiMetadata[T](
 ) {
   val httpMethods: List[OpenApiOperation[_]] = (gets: List[OpenApiOperation[_]]) ++ customBodyMethods ++ bodyMethods
 
+  // collect all tags
+  private lazy val openApiTags: List[Tag] = {
+    def createTags(method: OpenApiMethod[_]): List[Tag] =
```

Comment:
This method only creates 0 or 1 "tags". I'd suggest renaming and making it return `Opt[Tag]`, as it simplifies the code without any changes needed to callsites.